doit/action.py

```diff
@@ -214,15 +214,17 @@ def execute(self, out=None, err=None):
 
 
     def expand_action(self):
-        """expand action string using task meta informations
-        @returns (string) - expanded string after substitution
+        """Expand action using task meta informations if action is a string.
+        Convert `Path` elements to `str` if action is a list.
```

Comment:
better say exactly what you are doing (converting all elements to string), nothing specific about Path.